monai/bundle/scripts.py

```diff
@@ -1191,10 +1191,10 @@ def onnx_export(
 
 
 def ckpt_export(
-    net_id: str | None = None,
-    filepath: PathLike | None = None,
-    ckpt_file: str | None = None,
-    meta_file: str | Sequence[str] | None = None,
+    net_id: str | None = "network_def",
```

Comment:
I didn't add `deprecated_arg_default` here since no breaking change will be introduced, what do you think?
cc @wyli @ericspod

Reply:
sure, please update the docstring to clarify the default value. also, for os compatibility, the path should be `os.path.join('models', 'model.ts')` instead of `models/model.ts`?

Reply:
Good points, have updated docstring and the path. Thanks.